en/releases/1.13.md

```diff
@@ -0,0 +1,116 @@
+# Release 1.13
+
+- [Release 1.13](#release-1-13)
+  - [Pre Releases](#pre-releases)
+  - [Changes](#changes)
+    - [Common](#common)
+    - [Sensors](#sensors)
+    - [Hardware](#hardware)
+    - [MAVLink](#mavlink)
+    - [Commander](#commander)
+    - [Multicopter](#multicopter)
+    - [VTOL](#vtol)
+    - [Control](#control)
+    - [GPS](#gps)
+    - [NuttX](#nuttx)
+    - [CAN](#can)
+
+## Pre Releases
+
+- [RC 1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.13.0-rc1)
+- [Beta 1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.13.0-beta1)
+
+## Changes
+
+- Dynamic Control Allocation [PR#18776](https://github.com/PX4/PX4-Autopilot/pull/18776)
+
+  :::note
+    Turned OFF by default, use `SYS_CTRL_ALLOC` param to enable
```

Comment:
Can you link to this page instead? https://docs.px4.io/master/en/config/actuators.html

Reply:
I addressed this here: https://github.com/PX4/PX4-user_guide/pull/1913/commits/56aa02106799bb3954b5e28e4d21e9dfb01a2d55
Feel free to add changes there.